airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcDestinationHandler.kt

```diff
@@ -375,7 +376,10 @@ abstract class JdbcDestinationHandler<DestinationState>(
                     JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT
                 ) && isAirbyteExtractedAtColumnMatch(existingTable)) ||
                 !(existingTable.columns.containsKey(JavaBaseConstants.COLUMN_NAME_AB_META) &&
-                    isAirbyteMetaColumnMatch(existingTable))
+                    isAirbyteMetaColumnMatch(existingTable)) ||
+                !(existingTable.columns.containsKey(
+                    JavaBaseConstants.COLUMN_NAME_AB_GENERATION_ID
+                ) && isAirbyteMetaColumnMatch(existingTable))
```

Comment:
probably copypasta error, I think you want a new `isAirbyteGenerationColumnMatch` method?